src/include/stir/IO/GEHDF5Wrapper.h

```diff
@@ -76,37 +94,32 @@ class GEHDF5Wrapper
     Succeeded get_from_dataspace(std::streampos &current_offset,
                                  char* output);
 
-    Succeeded get_dataspace(const unsigned int current_id,
+    Succeeded get_singles(const unsigned int current_id,
```

Comment:
actually, can we reorder arguments here? Standard STIR convention is to have output arguments come first.

Reply:
Also, do we actually use both of the `get_singles` functions?

Reply:
We don't use both no. Want me to get rid of the one we dont?

Reply:
sure.